<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulador de Trens - Malha Hexagonal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_trem1">
    <property name="geometry"><rect><x>60</x><y>60</y><width>20</width><height>20</height></rect></property>
    <property name="styleSheet"><string>background: green; border: 1px solid black;</string></property>
    <property name="text"><string>T1</string></property>
    <property name="alignment"><set>Qt::AlignCenter</set></property>
   </widget>
   <widget class="QLabel" name="label_trem2">
    <property name="geometry"><rect><x>100</x><y>60</y><width>20</width><height>20</height></rect></property>
    <property name="styleSheet"><string>background: red; border: 1px solid black;</string></property>
    <property name="text"><string>T2</string></property>
    <property name="alignment"><set>Qt::AlignCenter</set></property>
   </widget>
   <widget class="QLabel" name="label_trem3">
    <property name="geometry"><rect><x>140</x><y>60</y><width>20</width><height>20</height></rect></property>
    <property name="styleSheet"><string>background: blue; border: 1px solid black;</string></property>
    <property name="text"><string>T3</string></property>
    <property name="alignment"><set>Qt::AlignCenter</set></property>
   </widget>
   <widget class="QLabel" name="label_trem4">
    <property name="geometry"><rect><x>180</x><y>60</y><width>20</width><height>20</height></rect></property>
    <property name="styleSheet"><string>background: orange; border: 1px solid black;</string></property>
    <property name="text"><string>T4</string></property>
    <property name="alignment"><set>Qt::AlignCenter</set></property>
   </widget>
   <widget class="QLabel" name="label_trem5">
    <property name="geometry"><rect><x>220</x><y>60</y><width>20</width><height>20</height></rect></property>
    <property name="styleSheet"><string>background: yellow; border: 1px solid black;</string></property>
    <property name="text"><string>T5</string></property>
    <property name="alignment"><set>Qt::AlignCenter</set></property>
   </widget>
   <widget class="QLabel" name="label_trem6">
    <property name="geometry"><rect><x>260</x><y>60</y><width>20</width><height>20</height></rect></property>
    <property name="styleSheet"><string>background: purple; border: 1px solid black; color: white;</string></property>
    <property name="text"><string>T6</string></property>
    <property name="alignment"><set>Qt::AlignCenter</set></property>
   </widget>

   <widget class="QSlider" name="horizontalSlider_1">
    <property name="geometry"><rect><x>50</x><y>400</y><width>100</width><height>20</height></rect></property>
    <property name="maximum"><number>200</number></property>
    <property name="value"><number>100</number></property>
    <property name="orientation"><enum>Qt::Horizontal</enum></property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_2">
    <property name="geometry"><rect><x>160</x><y>400</y><width>100</width><height>20</height></rect></property>
    <property name="maximum"><number>200</number></property>
    <property name="value"><number>100</number></property>
    <property name="orientation"><enum>Qt::Horizontal</enum></property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_3">
    <property name="geometry"><rect><x>270</x><y>400</y><width>100</width><height>20</height></rect></property>
    <property name="maximum"><number>200</number></property>
    <property name="value"><number>100</number></property>
    <property name="orientation"><enum>Qt::Horizontal</enum></property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_4">
    <property name="geometry"><rect><x>380</x><y>400</y><width>100</width><height>20</height></rect></property>
    <property name="maximum"><number>200</number></property>
    <property name="value"><number>100</number></property>
    <property name="orientation"><enum>Qt::Horizontal</enum></property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_5">
    <property name="geometry"><rect><x>490</x><y>400</y><width>100</width><height>20</height></rect></property>
    <property name="maximum"><number>200</number></property>
    <property name="value"><number>100</number></property>
    <property name="orientation"><enum>Qt::Horizontal</enum></property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_6">
    <property name="geometry"><rect><x>600</x><y>400</y><width>100</width><height>20</height></rect></property>
    <property name="maximum"><number>200</number></property>
    <property name="value"><number>100</number></property>
    <property name="orientation"><enum>Qt::Horizontal</enum></property>
   </widget>
   
   <widget class="QLabel" name="lbl_s1"><property name="geometry"><rect><x>80</x><y>425</y><width>40</width><height>20</height></rect></property><property name="text"><string>T1</string></property></widget>
   <widget class="QLabel" name="lbl_s2"><property name="geometry"><rect><x>190</x><y>425</y><width>40</width><height>20</height></rect></property><property name="text"><string>T2</string></property></widget>
   <widget class="QLabel" name="lbl_s3"><property name="geometry"><rect><x>300</x><y>425</y><width>40</width><height>20</height></rect></property><property name="text"><string>T3</string></property></widget>
   <widget class="QLabel" name="lbl_s4"><property name="geometry"><rect><x>410</x><y>425</y><width>40</width><height>20</height></rect></property><property name="text"><string>T4</string></property></widget>
   <widget class="QLabel" name="lbl_s5"><property name="geometry"><rect><x>520</x><y>425</y><width>40</width><height>20</height></rect></property><property name="text"><string>T5</string></property></widget>
   <widget class="QLabel" name="lbl_s6"><property name="geometry"><rect><x>630</x><y>425</y><width>40</width><height>20</height></rect></property><property name="text"><string>T6</string></property></widget>

  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>